.github/workflows/deploy_prod.yml

```diff
@@ -39,15 +39,15 @@ jobs:
           expo-password: ${{ secrets.EXPO_CLI_PROD_PASSWORD }}
           expo-cache: true
       - uses: rlespinasse/github-slug-action@v2.x
-      - name: Parse Major Version
-        id: majorVersion
+      - name: Generate Release Channel
+        id: releaseChannel
         run: |
-          MAJOR_VERSION=$(echo ${{ env.GITHUB_REF_SLUG }} | sed -r 's/\.[0-9]+\.[0-9]+$//')
-          echo "::set-output name=majorVersion::$MAJOR_VERSION"
+          RELEASE_CHANNEL=$(echo ${{ env.GITHUB_REF_SLUG }} | sed -r 's/\.[0-9]+\.[0-9]+$//')
```

Comment:
Just a minor thing, could you add a comment to showcase an example like the one you added on line 5?

Reply:
added on line 42 as didnt want to put it in run!